### PR TITLE
Add notebook for light curve simulation

### PR DESCRIPTION
**Pull request by AtreyeeS**
**Description**

This pull request adds a notebook to simulate light curves and do a joint fit using a temporal model.

**Dear reviewer**

I have put the entire thing in one notebook. It takes 9.6 s to run on my machine.

### CODE REVIEW

File: docs/tutorials/index.rst [hunk 1]
```diff
@@ -94,6 +95,7 @@ The following tutorials expose common analysis tasks.
 
 - `1D spectrum simulation <../notebooks/spectrum_simulation.html>`__ | *spectrum_simulation.ipynb*
 - `3D map simulation <../notebooks/simulate_3d.html>`__ | *simulate_3d.ipynb*
+- `Light curve simulation <../notebooks/light_curve_simulation.html>`__ | *light_curve_simulation.ipynb*
```
Comment (registerrier) [inline docs/tutorials/index.rst]: How about "simulating 1D observations of a time varying source"?

File: tutorials/light_curve_simulation.ipynb [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+    "\n",
+    "Frequently, studies of variable sources (eg: decaying GRB light curves, AGN flares, etc) require time variable simulations. For most use cases, generating an event list is an overkill, and it suffices to use binned simulations using a temporal model.\n",
+    "\n",
+    "**Objective: Simulate and fit a time decaying light curve of a source with CTA using the CTA 1DC response**\n",
+    "\n",
+    "## Proposed approach:\n",
+    "\n",
+    "We will simulate 10 spectral datasets within given time intervals (Good Time Intervals) following a given spectral (a power law) and temporal profile (an exponential decay, with a decay time of 6 hr ). These are then analysed using the light curve estimator to obtain flux points. Then, we re-fit the simulated datasets to reconstruct back the injected profiles.\n",
+    "\n",
+    "In summary, step are:\n",
```
Comment (registerrier) [inline tutorials/light_curve_simulation.ipynb]: the necessary steps are

File: tutorials/light_curve_simulation.ipynb [hunk 3]
```diff
@@ -0,0 +1,427 @@
...
+    "\n",
+    "- Choose observation parameters including a list of `gammapy.data.GTI`\n",
+    "- Define temporal and spectral models from :ref:model-gallery as per science case\n",
+    "- Perform the simulation (in 1D or 3D)\n",
+    "- Extract the light curve from the reduced dataset as shown in [light curve notebook](light_curve.ipynb)\n",
+    "- Fit the simulated datasets using a source model \n",
```
Comment (registerrier) [inline tutorials/light_curve_simulation.ipynb]: Maybe "Optionaly, we show here how to"

File: tutorials/light_curve_simulation.ipynb [hunk 4]
```diff
@@ -0,0 +1,427 @@
...
+   "outputs": [],
+   "source": [
+    "ax = lc_1d.plot(marker=\"o\", label=\"3D\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We have the reconstructed lightcurve at this point. Further standard analyis might involve modeling the temporal profiles with an analytical or theoretical model. You may do this using your favourite fiting package, one possible option being `curve_fit` inside `scipy.optimize`.\n",
```
Comment (registerrier) [inline tutorials/light_curve_simulation.ipynb]: "fitting"

File: tutorials/light_curve_simulation.ipynb [hunk 5]
```diff
@@ -0,0 +1,427 @@
...
+    "\n",
+    "In the next section, we show how to simulatenously fit the all datasets using a given temporal model. We will fit the amplitude, spectral index and the decay time scale. Note that `t_ref` should be fixed by default for the `ExpDecayTemporalModel`. For modelling and fitting more complex flares, you should attach the relevant model to each dataset. For more details on joint fiiting in gammapy, see [here](modeling.ipynb)."
```
Comment (registerrier) [inline tutorials/light_curve_simulation.ipynb]: "simultaneously"
"fitting"

File: tutorials/light_curve_simulation.ipynb [hunk 6]
```diff
@@ -0,0 +1,427 @@
...
+    "plt.xlabel(\"time since reference\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Extract the lightcurve\n",
+    "\n",
+    "This section uses standard light curve estimation tools for a 1D extraction. Only a spectral model needs to be defined in this case."
```
Comment (registerrier) [inline tutorials/light_curve_simulation.ipynb]: You might explain that the temporal component wouldn't be taken into account

**Comment by AtreyeeS** [comment]
Thanks @registerrier and @adonath for your comments - made the changes!

**Pull request merged**